### ISSUE TITLE
After install through the install.sh , I can't find the correspondent instructruction to start the app. Thanks.

### ISSUE THREAD

**Issue by njklo** [inventree/InvenTree#4652]
### Deployment Method

- [X] Installer
- [ ] Docker Development
- [ ] Docker Production
- [ ] Bare metal Development
- [ ] Bare metal Production
- [ ] Digital Ocean image
- [ ] Other (please provide a link `Steps to Reproduce`

### Describe the problem*

OS
Ubuntu 20.04 in virtualbox
----------
Install from
wget -q0 install.sh https://get.inventree.org && bash install.sh
----------
The doc I use to deploy it
https://docs.inventree.org/en/latest/start/production/
----------
The question I have:

In the doc above, it says to run

cd /home/InvenTree
source ./env/bin/activate

cd src/InvenTree
/home/inventree/env/bin/gunicorn -c gunicorn.conf.py InvenTree.wsgi -b 127.0.0.1:8000

But the directiory in my system is 
/opt/inventree/
and
/opt/inventree/InvenTree/
and 
/opt/inventree/Inventree/Inventree

Can you please tell me which deployment document should I follow?

Thank you very much! 

### Steps to Reproduce

0. Install from
wget -q0 install.sh https://get.inventree.org && bash install.sh

then 

1. the directory in my system is

/opt/inventree/
and
/opt/inventree/InvenTree/
and 
/opt/inventree/Inventree/Inventree

2. I followed the doc https://docs.inventree.org/en/latest/start/production/
and change the path 
from  /home/inventree/env/bin/gunicorn -c gunicorn.conf.py InvenTree.wsgi -b 127.0.0.1:8000
to  /opt/inventree/env/bin/gunicorn -c gunicorn.conf.py InvenTree.wsgi -b 127.0.0.1:8000
(I run it in directory of /opt/inventree/InvenTree because under this path there ia file gunicorn.conf.py, but I never find InvenTree.wsgi)

4. The command  _/opt/inventree/env/bin/gunicorn -c gunicorn.conf.py InvenTree.wsgi -b 127.0.0.1:8000_ can run but the app can not start and there are problems. The terminal notice that: INVENTREE_STATIC_ROOT not specified

### Relevant log output
$ sudo /opt/inventree/env/bin/gunicorn -c gunicorn.conf.py InvenTree.wsgi -b 127.0.0.1:8000
[2023-04-21 10:28:32 -0300] [23085] [INFO] Starting gunicorn 20.1.0
[2023-04-21 10:28:32 -0300] [23085] [INFO] Listening at: http://127.0.0.1:8000 (23085)
[2023-04-21 10:28:32 -0300] [23085] [INFO] Using worker: sync
[2023-04-21 10:28:32 -0300] [23087] [INFO] Booting worker with pid: 23087
[2023-04-21 10:28:32 -0300] [23088] [INFO] Booting worker with pid: 23088
[2023-04-21 10:28:32 -0300] [23089] [INFO] Booting worker with pid: 23089
[2023-04-21 10:28:32 -0300] [23090] [INFO] Booting worker with pid: 23090
[2023-04-21 10:28:32 -0300] [23091] [INFO] Booting worker with pid: 23091
[2023-04-21 10:28:33 -0300] [23087] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/wsgi.py", line 15, in <module>
    application = get_wsgi_application()  # pragma: no cover
  File "/opt/inventree/env/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/settings.py", line 107, in <module>
    STATIC_ROOT = config.get_static_dir()
  File "/opt/inventree/InvenTree/InvenTree/config.py", line 230, in get_static_dir
    raise FileNotFoundError('INVENTREE_STATIC_ROOT not specified')
FileNotFoundError: INVENTREE_STATIC_ROOT not specified
[2023-04-21 10:28:33 -0300] [23087] [INFO] Worker exiting (pid: 23087)
[2023-04-21 10:28:33 -0300] [23089] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/wsgi.py", line 15, in <module>
    application = get_wsgi_application()  # pragma: no cover
  File "/opt/inventree/env/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/settings.py", line 107, in <module>
    STATIC_ROOT = config.get_static_dir()
  File "/opt/inventree/InvenTree/InvenTree/config.py", line 230, in get_static_dir
    raise FileNotFoundError('INVENTREE_STATIC_ROOT not specified')
FileNotFoundError: INVENTREE_STATIC_ROOT not specified
[2023-04-21 10:28:33 -0300] [23089] [INFO] Worker exiting (pid: 23089)
[2023-04-21 10:28:33 -0300] [23088] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/wsgi.py", line 15, in <module>
    application = get_wsgi_application()  # pragma: no cover
  File "/opt/inventree/env/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/settings.py", line 107, in <module>
    STATIC_ROOT = config.get_static_dir()
  File "/opt/inventree/InvenTree/InvenTree/config.py", line 230, in get_static_dir
    raise FileNotFoundError('INVENTREE_STATIC_ROOT not specified')
FileNotFoundError: INVENTREE_STATIC_ROOT not specified
[2023-04-21 10:28:33 -0300] [23088] [INFO] Worker exiting (pid: 23088)
[2023-04-21 10:28:33 -0300] [23090] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/wsgi.py", line 15, in <module>
    application = get_wsgi_application()  # pragma: no cover
  File "/opt/inventree/env/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/settings.py", line 107, in <module>
    STATIC_ROOT = config.get_static_dir()
  File "/opt/inventree/InvenTree/InvenTree/config.py", line 230, in get_static_dir
    raise FileNotFoundError('INVENTREE_STATIC_ROOT not specified')
FileNotFoundError: INVENTREE_STATIC_ROOT not specified
[2023-04-21 10:28:33 -0300] [23090] [INFO] Worker exiting (pid: 23090)
[2023-04-21 10:28:33 -0300] [23091] [ERROR] Exception in worker process
Traceback (most recent call last):
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 589, in spawn_worker
    worker.init_process()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 134, in init_process
    self.load_wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/workers/base.py", line 146, in load_wsgi
    self.wsgi = self.app.wsgi()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 67, in wsgi
    self.callable = self.load()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 58, in load
    return self.load_wsgiapp()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 48, in load_wsgiapp
    return util.import_app(self.app_uri)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/util.py", line 359, in import_app
    mod = importlib.import_module(module)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/wsgi.py", line 15, in <module>
    application = get_wsgi_application()  # pragma: no cover
  File "/opt/inventree/env/lib/python3.8/site-packages/django/core/wsgi.py", line 12, in get_wsgi_application
    django.setup(set_prefix=False)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/__init__.py", line 19, in setup
    configure_logging(settings.LOGGING_CONFIG, settings.LOGGING)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 82, in __getattr__
    self._setup(name)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 69, in _setup
    self._wrapped = Settings(settings_module)
  File "/opt/inventree/env/lib/python3.8/site-packages/django/conf/__init__.py", line 170, in __init__
    mod = importlib.import_module(self.SETTINGS_MODULE)
  File "/usr/lib/python3.8/importlib/__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "<frozen importlib._bootstrap>", line 1014, in _gcd_import
  File "<frozen importlib._bootstrap>", line 991, in _find_and_load
  File "<frozen importlib._bootstrap>", line 975, in _find_and_load_unlocked
  File "<frozen importlib._bootstrap>", line 671, in _load_unlocked
  File "<frozen importlib._bootstrap_external>", line 848, in exec_module
  File "<frozen importlib._bootstrap>", line 219, in _call_with_frames_removed
  File "/opt/inventree/InvenTree/InvenTree/settings.py", line 107, in <module>
    STATIC_ROOT = config.get_static_dir()
  File "/opt/inventree/InvenTree/InvenTree/config.py", line 230, in get_static_dir
    raise FileNotFoundError('INVENTREE_STATIC_ROOT not specified')
FileNotFoundError: INVENTREE_STATIC_ROOT not specified
[2023-04-21 10:28:33 -0300] [23091] [INFO] Worker exiting (pid: 23091)
Traceback (most recent call last):
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 209, in run

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/opt/inventree/env/bin/gunicorn", line 8, in <module>
    sys.exit(run())
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/wsgiapp.py", line 67, in run
    WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]").run()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 231, in run
    super().run()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/app/base.py", line 72, in run
    Arbiter(self).run()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 229, in run
    self.halt(reason=inst.reason, exit_status=inst.exit_status)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 342, in halt
    self.stop()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 393, in stop
    time.sleep(0.1)
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
    self.reap_workers()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 530, in reap_workers
    self.log.warning(
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/glogging.py", line 261, in warning
    self.error_log.warning(msg, *args, **kwargs)
  File "/usr/lib/python3.8/logging/__init__.py", line 1458, in warning
    self._log(WARNING, msg, args, **kwargs)
  File "/usr/lib/python3.8/logging/__init__.py", line 1587, in _log
    record = self.makeRecord(self.name, level, fn, lno, msg, args,
  File "/usr/lib/python3.8/logging/__init__.py", line 1556, in makeRecord
    rv = _logRecordFactory(name, level, fn, lno, msg, args, exc_info, func,
  File "/usr/lib/python3.8/logging/__init__.py", line 293, in __init__
    ct = time.time()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 242, in handle_chld
    self.reap_workers()
  File "/opt/inventree/env/lib/python3.8/site-packages/gunicorn/arbiter.py", line 525, in reap_workers
    raise HaltServer(reason, self.WORKER_BOOT_ERROR)
gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>

_No response_

**Comment by matmair** [comment]
@njklo did the suggestion help?

**Comment by njklo** [comment]
> you need to configure INVENTREE_STATIC_ROOT en INVENTREE_MEDIA_ROOT in the config.yaml file
@DS1705 

> @njklo did the suggestion help?

Thank you for your kind guide very much! The Command line installation is too hard for me because I don't have much knowledge about it. 
So I didn't continue the command line installation in my virtual box's Ubuntu, but I changed to another installation method - docker. 

I just finished reading the docker beginner document in these days and I'm trying to install Inventree following this document now - https://docs.inventree.org/en/latest/start/docker_prod/  

I'm in the middle of the process of docker-compose installation now. I hope I can complete the installation later.

Thank you a lot!
